README.md

```diff
@@ -5,31 +5,31 @@ Before you run MATLAB&reg; code and Simulink&reg; models on a [GitHub&reg;-hoste
 The **Setup MATLAB** action is not supported on [self-hosted](https://docs.github.com/en/free-pro-team@latest/actions/hosting-your-own-runners/about-self-hosted-runners) runners. Currently, it is available only for public projects. It does not set up transformation products, such as MATLAB Coder&trade; and MATLAB Compiler&trade;.
 
 ## Usage Examples
-Once you set up MATLAB, you can use the runner to execute MATLAB scripts, functions, or statements. You also can use the runner to execute MATLAB and Simulink tests and generate artifacts. To execute code on the runner, include the [Run MATLAB Command](https://github.com/matlab-actions/run-command/) or [Run MATLAB Tests](https://github.com/matlab-actions/run-tests/) actions in your workflow.
+Once you set up MATLAB, you can build and test your MATLAB project as part of your workflow. To execute code on the runner, include the [Run MATLAB Build](https://github.com/matlab-actions/run-build/), [Run MATLAB Tests](https://github.com/matlab-actions/run-tests/), or [Run MATLAB Command](https://github.com/matlab-actions/run-command/) actions in your workflow.
 
-### Run MATLAB Script on GitHub-Hosted Runner
-Set up a GitHub-hosted runner to run the commands in a file named `myscript.m` in the root of your repository. To run the script, include the [Run MATLAB Command](https://github.com/matlab-actions/run-command/) action in your workflow.
+### Run MATLAB Build on GitHub-Hosted Runner
+Set up a GitHub-hosted runner to run a specific task and its depended-on tasks that are specified in a file named `buildfile.m` in the root of your repository. To run tasks using the MATLAB build tool, include the [Run MATLAB Build](https://github.com/matlab-actions/run-build/) action in your workflow. This action is supported in MATLAB R2022b and later.
 
 ```yaml
-name: Run MATLAB Script on GitHub-Hosted Runner
+name: Run MATLAB Build on GitHub-Hosted Runner
 on: [push]
 jobs:
   my-job:
-    name: Run MATLAB Script
+    name: Run MATLAB Build
     runs-on: ubuntu-latest
     steps:
       - name: Check out repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: Set up MATLAB
         uses: matlab-actions/setup-matlab@v1
-      - name: Run script
-        uses: matlab-actions/run-command@v1
+      - name: Run build
+        uses: matlab-actions/run-build@v1
```

Comment:
@v0

Maybe run-build should've (or should) be released as v1.0.0, but currently it's v0.1.0

Reply:
Oh dang. Can we just burn v0 and push up v1?

Reply:
Yeah, I think should be fine. Do we want to delete that release or just release a new v1.0.0?

Reply:
We can keep v0 and just push up v1 on top

Reply:
Sounds good. I just bumped to v1. It should be up now